client/components/domains/register-domain-step/index.jsx

```diff
@@ -134,7 +134,7 @@ const RegisterDomainStep = React.createClass( {
 			analyticsSection: 'domains',
 			onSave: noop,
 			onAddMapping: noop,
-			onAddDomain: noop,
+			onAddDomain: noop
```

Comment:
Actually, I'd consider it very necessary - it makes adding more properties here easier (smaller/simpler diffs)